src/main/java/io/papermc/lib/environments/Environment.java

```diff
@@ -112,6 +119,11 @@ public BlockStateSnapshotResult getBlockState(Block block, boolean useSnapshot)
     public CompletableFuture<Location> getBedSpawnLocationAsync(Player player, boolean isUrgent) {
         return bedSpawnLocationHandler.getBedSpawnLocationAsync(player, isUrgent);
     }
+    
+    @Nullable
+    public OfflinePlayer getOfflinePlayer(@Nonnull String name, boolean makeWebRequest) {
```

Comment:
probably should be renamed to getOfflinePlayerCached

Reply:
Not too sure about this change, `getOfflinePlayerCached` would imply the value to always be cached and the Player can sometimes be retrieved via the uuid lookup, so I think `getOfflinePlayer` is - although not ideal - a better choice here.

Reply:
True, usually the methods are named after the effect on Paper so that's why I suggested that but since your parameter allows for them to sometimes do uuid lookups I see the problem.

Reply:
You missed a keyword "If" in your statement @JRoy 

getOfflinePlayerIfCached would be the ideal name to match the Paper API name which has a very different interpretation than getOfflinePlayerCached

Reply:
Alright then, renamed to `getOfflinePlayerIfCached` to match with the Paper API method.